project/AppDependencies.scala

```diff
@@ -7,14 +7,14 @@ object AppDependencies {
   val compile = Seq(
     ws,
     "uk.gov.hmrc"                   %% "bootstrap-backend-play-28"  % boostrapVersion,
-    "com.fasterxml.jackson.module"  %% "jackson-module-scala"       % "2.14.2"
+    "com.fasterxml.jackson.module"  %% "jackson-module-scala"       % "2.15.2"
```

Comment:
what happens if you now remove this dependency - does the service still work?

Reply:
Then we get the vulnerable library back in use

Reply:
I did look at taking it out, but we don't want to

Reply:
Otherwise it compiles and tests pass